api/agent/agent.go

```diff
@@ -520,7 +523,9 @@ func (a *agent) waitHot(ctx context.Context, call *call, caller *slotCaller) (Sl
 	// 1) if we can get a slot immediately, grab it.
 	// 2) if we don't, send a signaller every x msecs until we do.
 
-	sleep := 1 * time.Microsecond // pad, so time.After doesn't send immediately
+	timer := common.NewTimer(1 * time.Microsecond) // pad, so time.After doesn't send immediately
```

Comment:
Nit: `time.After` is no longer used here, making this comment slightly out of date.